packages/common/src/store/ui/modals/publish-content-modal.ts

```diff
@@ -0,0 +1,25 @@
+import { ID } from '~/models/Identifiers'
+import { Nullable } from '~/utils/typeUtils'
+
+import { createModal } from './createModal'
+
+export type PublishContentModalState = {
+  contentId: Nullable<ID>
+  contentType: Nullable<'playlist' | 'album' | 'track'>
```

Comment:
we should pull out this type somewhere eventually huh? feel like we've been using this a lot. but no need for this PR.